feat(docker/java): add workaround to limit releases to java LTS versions (#18007)
renovatebot/renovate: docs/usage/java.md

> ## LTS Releases

Reviewer: ```suggestion
## LTS releases
```

> The `config:base` preset limits Renovate update to Java runtimes to those which are LTS versions. This is handled by the workaround `workarounds:javaLTSVersions`.

Reviewer: ```suggestion
The `config:base` preset includes the `workarounds:javaLTSVersions` preset.
The workaround limits Renovate to upgrade to LTS versions of the Java runtime only.
```

> If you wish to have all major updates suggested then you can add the following `ignorePreset` rule:

Reviewer: ```suggestion
If you want Renovate to offer all `major` Java updates then add `workarounds:javaLTSVersions` to the `ignorePreset` array:
```
